addons/l10n_mt_pos/models/compliance_letter.py

```diff
@@ -0,0 +1,25 @@
+from odoo import models, fields, api, release
+from datetime import datetime
+
+class ComplianceLetter(models.Model):
+    _name = 'compliance.letter'
```

Comment:
Is there a reason we need a new model, could we not simply add a report on the company?